### ISSUE TITLE
False Positive(s) on json-simple-1.1.1.jar

### ISSUE THREAD

**Issue by mprins** [dependency-check/DependencyCheck#2792]
Two false positives are reported on json-simple-1.1.1.jar (CVE-2020-10663 and CVE-2020-7712) using the DependencyCheck 6.0.0 Maven plugin

1. `cpe:2.3:a:json_project:json:*:*:*:*:*:ruby:*:*` versions up to (including) 2.2.0  (this is a Ruby library, not a Java lib)
2. `cpe:2.3:a:json_project:json:*:*:*:*:*:node.js:*:*` versions up to (excluding) 10.0.0 (this is a Node library, not a Java lib)

```xml
<dependency>
    <groupId>com.googlecode.json-simple</groupId>
    <artifactId>json-simple</artifactId>
    <version>1.1.1</version>
</dependency>
```

**Comment by jeremylong** [comment]
Just tested with 6.0.2 and this appears to be resolved.

**Comment by volkert-fastned** [comment]
@jeremylong Unfortunately, I just encountered one of these false positives (CVE-2020-7712) when I upgraded the DependencyCheck Gradle plugin from 5.3.2 to 6.0.2. So apparently it isn't entirely fixed in 6.0.2, at least not in the Gradle plugin.
